# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/iptraf/pedantic-setup.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Setup.orig	2005-05-09 23:10:54.000000000 +0200
+++ ./Setup	2005-05-09 23:11:32.000000000 +0200
@@ -26,9 +26,9 @@
     if [ -f src/iptraf.c ]; then
         echo ">>>>>> COMPILING IPTRAF $VERSION FROM SUPPLIED SOURCE CODE"
         echo
-        /usr/bin/make -C src clean
-        /usr/bin/make -C support clean
-        /usr/bin/make -C src all
+        /usr/bin/make -C src clean	|| exit 1
+        /usr/bin/make -C support clean	|| exit 1
+        /usr/bin/make -C src all	|| exit 1
     else
         echo "*** ERROR: Unable to locate source files."
         echo "*** If this is a binary-only distribution of IPTraf, try running $0"
@@ -40,5 +40,5 @@
     fi
 fi
 
-/usr/bin/make -C src install
+/usr/bin/make -C src install || exit 1
 
